nncf/common/tensor_statistics/statistics.py

```diff
@@ -41,6 +41,25 @@ def __eq__(self, other: 'MinMaxTensorStatistic') -> bool:
                self.tensor_eq(self.max_values, other.max_values)
 
 
+class MeanTensorStatistic(TensorStatistic):
+    """
+    Base class for the statistics that collects as mean per-axis
```

Comment:
Some words seem to be missing there

Reply:
Can you be more specific? Don't hesitate to make proposals. It's much more efficient. We are a little in a hurry with this and follow-up PRs.